members/FMNS420b9c88.yaml

```diff
@@ -65,7 +65,7 @@ contact_form:
           Senator: Senator
       - name: topic
         selector: select#edit-submitted-message-details-topic
-        value: $TOPIC
+        value: "$TOPIC"
```

Comment:
for the future - the value doesn't need to be in quotes

Reply:
yeap. Since most of the selectors above were inside quotes and I just thought about making the styles match for the sake of consistency